Design Patterns/Abstract Factory Pattern/src/GPUs/GpuBrand.java

```diff
@@ -0,0 +1,5 @@
+package GPUs;
```

Comment:
please follow naming convention for packaging: like: com.exalt.training.designpattern

please apply it for all